modules/logging-forwarding-azure.adoc

```diff
@@ -0,0 +1,139 @@
++
+[source,text]
+----
+Get-AzOperationalInsightsWorkspaceSharedKey -ResourceGroupName "<resource_name>" -Name "<workspace_name>”
+----
++
+
+. Create or edit your `ClusterLogForwarder` CR using the template matching your log selection.
++
+.Forward all logs
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+  - name: azure-monitor
+    type: azureMonitor
+    azureMonitor:
+      customerId: my-customer-id # <1>
+      logType: my_log_type # <2>
+    secret:
+       name: my-secret
+  pipelines:
+  - name: app-pipelinehttps://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-platform-logs
+    inputRefs:
+    - application
+    outputRefs:
+    - azure-monitor
+----
+<1> Unique identifier for the Log Analytics workspace. Required field.
+<2> link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell#record-type-and-properties[Azure record type] of the data being submitted. May only contain letters, numbers, and underscores (_), and may not exceed 100 characters.
++
+.Forward application and infrastructure logs
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+  - name: azure-monitor-app
+    type: azureMonitor
+    azureMonitor:
+      customerId: my-customer-id
+      logType: application_log # <1>
+    secret:
+      name: my-secret
+  - name: azure-monitor-infra
+    type: azureMonitor
+    azureMonitor:
+      customerId: my-customer-id
+      logType: infra_log #
+    secret:
+      name: my-secret
+  pipelines:
+    - name: app-pipeline
+      inputRefs:
+      - application
+      outputRefs:
+      - azure-monitor-app
+    - name: infra-pipeline
+      inputRefs:
+      - infrastructure
+      outputRefs:
+      - azure-monitor-infra
+----
+<1> link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell#record-type-and-properties[Azure record type] of the data being submitted. May only contain letters, numbers, and underscores (_), and may not exceed 100 characters.
++
+.Advanced configuration options
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+  - name: azure-monitor
+    type: azureMonitor
+    azureMonitor:
+      customerId: my-customer-id
+      logType: my_log_type
+      azureResourceId: "/subscriptions/111111111" # <1>
+      host: "ods.opinsights.azure.acme" # <2>
+    secret:
+       name: my-secret
+  pipelines:
+   - name: app-pipeline
+    inputRefs:
+    - application
+    outputRefs:
+    - azure-monitor
+----
+<1> Resource ID of the Azure resource the data should be associated with. Optional field.
+<2> Alternative host for dedicated Azure regions. Optional field.
```

Comment:
The default value is ods.opinsights.azure.com. For azure goverment:  it is ods.opinsights.azure.us